lsp-modeline.el

```diff
@@ -236,7 +252,9 @@ The `:global' workspace is global one.")
   (cl-labels ((calc-modeline ()
                              (let ((str (lsp-modeline-diagnostics-statistics)))
                                (if (string-empty-p str) ""
-                                 (concat " " str)))))
+                                 (concat " " str
```

Comment:
You will need to check for code action mode line too, and workspace status, download status mode line ...
User can customize and change order of those in `global-mode-string`.
Speaking of that, wouldn't it better to add an option for trailing front/behind space (maybe via a format string) instead of manually adding minor modes to `lsp-modeline--overlap-minor-modes`

Reply:
Got it! I have thought of that too. I guess that would be another solution? :)